From 357c6deafabd210cc140d5d6a5b22687cd609878 Mon Sep 17 00:00:00 2001
From: Andrey Konovalov <andrey.konovalov@linaro.org>
Date: Thu, 15 Nov 2012 00:22:35 +0400
Subject: [PATCH] Enable the options needed to test CPU hotplug latency

Enable CONFIG_HOTPLUG in linaro-base.conf as all Linaro kernels should
have it. Enabling CONFIG_DEBUG_FS and CONFIG_FUNCTION_TRACER add no
runtime slowdown if not used, so having them in linaro-base.conf is OK
too. Now that CONFIG_HOTPLUG and CONFIG_DEBUG_FS are in linaro-base.conf,
remove them from ubuntu.conf.

Signed-off-by: Andrey Konovalov <andrey.konovalov@linaro.org>
---
 linaro/configs/linaro-base.conf | 3 +++
 linaro/configs/ubuntu.conf      | 2 --
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/linaro/configs/linaro-base.conf b/linaro/configs/linaro-base.conf
index 9269c8e..eaf0196 100644
--- a/linaro/configs/linaro-base.conf
+++ b/linaro/configs/linaro-base.conf
@@ -7,6 +7,7 @@ CONFIG_IKCONFIG_PROC=y
 CONFIG_LOG_BUF_SHIFT=16
 CONFIG_BLK_DEV_INITRD=y
 CONFIG_EMBEDDED=y
+CONFIG_HOTPLUG=y
 CONFIG_PERF_EVENTS=y
 CONFIG_SLAB=y
 CONFIG_PROFILING=y
@@ -73,6 +74,7 @@ CONFIG_NLS_CODEPAGE_437=y
 CONFIG_NLS_ISO8859_1=y
 CONFIG_PRINTK_TIME=y
 CONFIG_MAGIC_SYSRQ=y
+CONFIG_DEBUG_FS=y
 CONFIG_SCHEDSTATS=y
 CONFIG_TIMER_STATS=y
 CONFIG_PROVE_LOCKING=y
@@ -83,4 +85,5 @@ CONFIG_CRC_T10DIF=y
 CONFIG_CRC_ITU_T=y
 CONFIG_CRC7=y
 CONFIG_HW_PERF_EVENTS=y
+CONFIG_FUNCTION_TRACER=y
 CONFIG_ENABLE_DEFAULT_TRACERS=y
diff --git a/linaro/configs/ubuntu.conf b/linaro/configs/ubuntu.conf
index 88e58df..fafabe1 100644
--- a/linaro/configs/ubuntu.conf
+++ b/linaro/configs/ubuntu.conf
@@ -45,7 +45,6 @@ CONFIG_UID16=y
 CONFIG_SYSCTL_SYSCALL=y
 CONFIG_KALLSYMS=y
 CONFIG_KALLSYMS_ALL=y
-CONFIG_HOTPLUG=y
 CONFIG_PRINTK=y
 CONFIG_BUG=y
 CONFIG_ELF_CORE=y
@@ -1943,7 +1942,6 @@ CONFIG_DLM=m
 CONFIG_DEFAULT_MESSAGE_LOGLEVEL=4
 CONFIG_FRAME_WARN=1024
 CONFIG_UNUSED_SYMBOLS=y
-CONFIG_DEBUG_FS=y
 CONFIG_DEBUG_KERNEL=y
 CONFIG_LOCKUP_DETECTOR=y
 CONFIG_BOOTPARAM_HARDLOCKUP_PANIC_VALUE=0
-- 
1.8.0

